src/classes/AFFL_MultiRecordType_TEST.cls

```diff
@@ -1109,188 +1443,396 @@ public with sharing class AFFL_MultiRecordType_TEST {
-           primaryFieldsList.add('Primary Sports Organization');      
+        primaryFieldsList.add(bizOrgContactFieldApiName);
+        primaryFieldsList.add(householdContactFieldLabel);
 
         Test.startTest();
-           AFFL_MultiRecordTypeMapper afflMapper = new AFFL_MultiRecordTypeMapper();
-           List<String> validPrimaryFieldList = afflMapper.getValidPrimaryAfflFields(primaryFieldsList);
+            AFFL_MultiRecordTypeMapper afflMapper = new AFFL_MultiRecordTypeMapper();
+            List<String> validPrimaryFieldList = afflMapper.getValidPrimaryAfflFields(primaryFieldsList);
         Test.stopTest();
 
-        // Ensure List has all 6 mappings
-        System.assertEquals(6, validPrimaryFieldList.size());
+        // Ensure List has all mappings
+        System.assertEquals(wrapperList.size(), validPrimaryFieldList.size());
 
     }
     
     // Positive test for getPrimaryAffiliationLookupAPI method of AFFL_MultiRecordTypeMapper using API name
     @isTest
     private static void getPrimaryAffiliationLookupAPIPoitiveAPITest() {
+        AFFL_MultiRecordType_TEST.setupAndGetMappingsByNameAndLabel();
 
-        AFFL_MultiRecordType_TEST.setup();
-        
-        String namespace = UTIL_Namespace.getNamespace();
-        String prefix = (String.isNotBlank(namespace)) ? namespace + '__' : '';
-        String lookupAPI = prefix + 'Primary_Organization__c';
+        String bizOrgContactFieldApiName =
+            AFFL_MultiRecordType_TEST.getBusinessOrganizationFieldApiName();
 
         Test.startTest();
             AFFL_MultiRecordTypeMapper afflMapper = new AFFL_MultiRecordTypeMapper();
-            String primaryAfflLookupAPIName = afflMapper.getPrimaryAffiliationLookupAPI(lookupAPI);
+            String primaryAfflLookupAPIName =
+                afflMapper.getPrimaryAffiliationLookupAPI(bizOrgContactFieldApiName);
         Test.stopTest();
 
         
-        System.assertEquals(prefix + 'Primary_Organization__c', primaryAfflLookupAPIName);
-       
+        System.assertEquals(bizOrgContactFieldApiName, primaryAfflLookupAPIName);
     }
     
     // Negative test for getPrimaryAffiliationLookupAPI method of AFFL_MultiRecordTypeMapper
     @isTest
     private static void getPrimaryAffiliationLookupAPINegativeTest() {
 
-        AFFL_MultiRecordType_TEST.setup();
+        AFFL_MultiRecordType_TEST.setupAndGetMappingsByNameAndLabel();
 
         Test.startTest();
             AFFL_MultiRecordTypeMapper afflMapper = new AFFL_MultiRecordTypeMapper();
             String primaryAfflLookupAPIName = afflMapper.getPrimaryAffiliationLookupAPI('Test1234');
         Test.stopTest();
 
         System.assertEquals(null, primaryAfflLookupAPIName);
-       
+
+    }
+
+/**************************************************************************
+* INNER CLASS MODELS
+**************************************************************************/
+
+private abstract class AffiliationMappingWrapper {
```

Comment:
Curious what's the reason why we want to have this new inner class in unit test. We do have affiliationWrapper class already?

Reply:
Per @MatthewBlanski, we will eventually move this to a separate class.

Reply:
As additional context, our current wrapper, the Affiliation Mapper, is a pseudo-service that has numerous static collections (such as lists of strings, etc), which does not provide the necessary individual behavior to handle the difference in functionality.

I'm hoping that during our contact refactor we look at replacing that class with something similar to these classes with more oomph so they appropriately represent what we 'know' about the mappings provided.